2007-07-10  Eric Blake <ebb9-at-byu.net>

	* lib/version-etc.c (version_etc_va): Default to GPLv3+.

--- gnulib/lib/version-etc.c
+++ gnulib/lib/version-etc.c
@@ -67,7 +67,7 @@ version_etc_va (FILE *stream,
 
   fputs (_("\
 \n\
-License GPLv2+: GNU GPL version 2 or later <http://gnu.org/licenses/gpl.html>\n\
+License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>\n\
 This is free software: you are free to change and redistribute it.\n\
 There is NO WARRANTY, to the extent permitted by law.\n\
 \n\
